force-app/main/default/translations/es.translation-meta.xml

```diff
@@ -4,21 +4,33 @@
         <name>Add_Service_Delivery</name>
         <label>Agregar entrega de servicio</label>
     </customLabels>
+    <customLabels>
+        <name>Anonymous</name>
```

Comment:
@AndersonTarren - just a note that I do not believe this label is in use anywhere.

Reply:
@AndersonTarren - Disregard. Bharathi had me double check and I was staring right at it but wasn't seeing it. One of those days.